defaults/main.yml

```diff
@@ -29,7 +29,7 @@ docker_compose_package_state: present
 
 # Docker Compose options.
 docker_install_compose: false
-docker_compose_version: "v2.11.1"
+docker_compose_version: "v2.29.2"
```

Comment:
Can you also update in the README?

Reply:
Sorry, I oversaw it. Aligned the version in the README.md.